src/inventory_ui.h

```diff
@@ -148,6 +152,44 @@ class inventory_entry
 
 };
 
+class group_entry: public inventory_entry
+{
+    public:
+        bool expanded = true;
+
+        group_entry();
+
+        group_entry( const item_category *custom_category ) :
+            inventory_entry( custom_category ) {};
+
+        group_entry( const inventory_entry &entry, const item_category *custom_category ) :
+            inventory_entry( entry, custom_category ) {};
+
+        group_entry( const std::vector<item_location> &locations,
+                     const item_category *custom_category = nullptr, bool enabled = true,
+                     const size_t chosen_count = 0 ) :
+            inventory_entry( locations, custom_category, enabled, chosen_count ) {};
+
+
+        // TODO: override, override, override
+
+        /**
+         * Group entry is not an item.
+         */
+        bool is_item() const {
+            return false;
+        }
```

Comment:
I'm thinking, don't allow it, unless asked for it. But, this throws me into a philosophical infinite loop.